pkg/gen/input/makefile/internal/file/Makefile.gen.cluster_app.mk.template

```diff
@@ -0,0 +1,21 @@
+{{ .Header }}
+
+
+VALUES=$(shell find ./helm -maxdepth 2 -name values.yaml)
+VALUES_SCHEMA=$(shell find ./helm -maxdepth 2 -name values.schema.json)
+
+.PHONY: normalize-schema
+normalize-schema:
+	go install github.com/giantswarm/schemalint@latest
```

Comment:
Also here. I think using `latest` here and later in this file is not a good idea

Reply:
Other makefiles use `docker run` instead of executing `go install`. Debatable if running a container is more portable/better/anything.

Reply:
I replaced `latest` with a major floating tag.
Concerning `docker run`: I will leave this as-is for now. If someone complains, it will still be possible to adapt.